### PR TITLE
chore: enable arm64 builds

### PR DESCRIPTION
**Pull request by cjdcordeiro**
Add arm64 builds for every python rock

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -5,11 +5,12 @@ on: [push, pull_request]
 jobs:
   test:
     name: Build and test
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runner }}
     strategy:
       fail-fast: false
       matrix:
         version: ["3.8", "3.10", "3.11", "3.12"]
+        runner: ["X64", "ARM64"]
```
Comment (rebornplusplus) [inline .github/workflows/tests.yml:13]: Is `X64` the same as `amd64`? If so, it would be nice to print that (`amd64`) out instead of `X64`.

**Review by rebornplusplus — COMMENTED** [review]
Looks good to me. Only one question.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (cjdcordeiro) [reply .github/workflows/tests.yml:13]: this is a GH label that only the runners' admin can change. by default GH uses x64. 

it feels unnecessary to add more complexity to the workflow just to translate that into amd64

Comment (rebornplusplus) [reply .github/workflows/tests.yml:13]: Alright.

**Review by rebornplusplus — APPROVED** [review]

**Review by zhijie-yang — APPROVED** [review]
LGTM, thanks!

**Pull request merged**